get-started/06_bind_mounts.md

```diff
@@ -1,121 +1,223 @@
-all of the build tools and environments installed. With a single `docker run` command, the dev environment is pulled and ready
-to go. We'll talk about Docker Compose in a future step, as this will help simplify our commands (we're already getting a lot
-of flags).
+1. Make sure you don't have any `getting-started` containers currently running.
+
+2. Run the following command from the `getting-started/app` directory.
+
+   If you are using an Mac or Linux device, then use the following command.
+
+   ```console
+   $ docker run -dp 3000:3000 \
+       -w /app --mount type=bind,src="$(pwd)",target=/app \
+       node:18-alpine \
+       sh -c "yarn install && yarn run dev"
+   ```
+
+   If you are using Windows, then use the following command in PowerShell.
+
+   ```powershell
+   $ docker run -dp 3000:3000 `
+       -w /app --mount type=bind,src="$(pwd)",target=/app `
+       node:18-alpine `
+       sh -c "yarn install && yarn run dev"
+   ```
+
+   - `-dp 3000:3000` - same as before. Run in detached (background) mode and
+     create a port mapping
+   - `-w /app` - sets the "working directory" or the current directory that the
+     command will run from
+   - `--mount type=bind,src="$(pwd)",target=/app` - bind mount the current
+     directory from the host into the `/app` directory in the container
+   - `node:18-alpine` - the image to use. Note that this is the base image for
+     our app from the Dockerfile
+   - `sh -c "yarn install && yarn run dev"` - the command. We're starting a
+     shell using `sh` (alpine doesn't have `bash`) and running `yarn install` to
+     install packages and then running `yarn run dev` to start the development
+     server. If we look in the `package.json`, we'll see that the `dev` script
+     starts `nodemon`.
+
+3. You can watch the logs using `docker logs`. You'll know you're ready to go
+   when you see this:
+
+   ```console
+   $ docker logs -f <container-id>
+   nodemon src/index.js
+   [nodemon] 2.0.20
+   [nodemon] to restart at any time, enter `rs`
+   [nodemon] watching dir(s): *.*
+   [nodemon] starting `node src/index.js`
+   Using sqlite database at /etc/todos/todo.db
+   Listening on port 3000
+   ```
+
+   When you're done watching the logs, exit out by hitting `Ctrl`+`C`.
+
+4. Now, make a change to the app. In the `src/static/js/app.js` file, on line
+   109, change the "Add Item" button to simply say "Add":
+
+   ```diff
+   - {submitting ? 'Adding...' : 'Add Item'}
+   + {submitting ? 'Adding...' : 'Add'}
+   ```
+
+   Save the file.
+
+5. Refresh the page in your web browser, and you should see the change reflected
+   almost immediately. It might take a few seconds for the Node server to
+   restart. If you get an error, try refreshing after a few seconds.
+
+   ![Screenshot of updated label for Add button](images/updated-add-button.png){:
+   style="width:75%;"} {: .text-center }
```

Comment:
{: .text-center } appears as text in the preview. I'm unsure why, maybe it's the space before.